src/wallet.ts

```diff
@@ -907,7 +975,14 @@ export class Wallet {
       : new BN(options?.attachedDeposit)
 
     // @ts-ignore: method does not exist on Contract type
-    await contract.create_store(storeData, gas, attachedDeposit)
+    await contract.create_store({
```

Comment:
It would be a lot nicer if the method did exist on contract type :) Like @tifrel idea of generating these interfaces from Rust code

Reply:
I think part of this limitation comes from using the `Contract` type from `near-api-js`, but 100% on figuring out a solution. Generating from the source would be awesome.